### DISCUSSION TITLE
Added: LDM notes for 2018-09-10

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#1878] [General]
[Notes for September 10th, 2018](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-09-10.md)

# Agenda

1. Nullability of constraints in inheritance and interface implementation
2. Static local functions

**Comment by CyrusNajmabadi** [comment]
Notes don't link to anything.

**Comment by agocke** [comment]
Woops, fixed.

**Comment by TheUnlocked** [comment]
So for the static local functions, do they capture `this` assuming their enclosing method is non-static, or are they completely divorced from the instance like a non-local static method is? Maybe I'm just getting lost in all of the jargon, but I can't see the answer to that.

**Comment by agocke** [comment]
@TheUnlocked No capturing, including `this`.
